### PR TITLE
Removed weather controller

### PR DESCRIPTION
**Pull request by Rapkup**

### CODE REVIEW

File: LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -1,33 +0,0 @@
-using Microsoft.AspNetCore.Mvc;
-
-namespace LX.StaffScheduler.Api.Controllers
-{
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs:7]: Please create new contoller related to your further tasks but without any actions toward to save /Controllers directory.

File: LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -1,33 +0,0 @@
...
-
-        private readonly ILogger<WeatherForecastController> _logger;
-
-        public WeatherForecastController(ILogger<WeatherForecastController> logger)
-        {
-            _logger = logger;
-        }
-
-        [HttpGet(Name = "GetWeatherForecast")]
-        public IEnumerable<WeatherForecast> Get()
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.Api/Controllers/WeatherForecastController.cs:22]: In assembly you have a file of class WeatherForecast so please remove it along with unnecessary controller

File: LX.StaffScheduler.Api/appsettings.Development.json [hunk 3]
```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=HOME-PC;Initial Catalog=CafeDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=RAPKUPICH;Initial Catalog=CafeDB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```
Comment (IvanLukyanau) [inline LX.StaffScheduler.Api/appsettings.Development.json:9]: can be omitted toward to leave these settings of connection not modified toward to exclude the json file from diff

**Review by IvanLukyanau — CHANGES_REQUESTED** [review]

**Review by IvanLukyanau — APPROVED** [review]

**Review by Karisha04 — APPROVED** [review]
PERFECT!!!!!!!!!

**Pull request merged**